.changeset/large-hotels-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-node': patch
+---
+
+Create an auth cookie middleware utility to be used by plugins setting a `user-cookie` policy. The middleware adds well known endpoints to issue and delete user cookies.
```

Comment:
This needs an update now. In general I also think it's best to mention all new exports that have been added to the package unless the list is very long.

Reply:
Removed